package.json

```diff
@@ -108,28 +103,25 @@
     "next-compose-plugins": "^2.2.1",
     "next-transpile-modules": "^8.0.0",
     "prettier": "^2.4.1",
-    "sass": "^1.32.8",
-    "sass-loader": "^12.4.0",
-    "semantic-release": "^17.4.2",
-    "typescript": "^4.5.4"
+    "semantic-release": "^19.0.2",
```

Comment:
Updated this to resolve security alerts – the breaking changes were dropping support for old Node versions.